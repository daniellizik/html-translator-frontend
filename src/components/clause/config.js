export const QUERY = {
  targets: {
    NODE_NAME: {
      rules: ['EQUALS','NOT_EQUALS','LIKE','NOT_LIKE']
    },
    TEXT: {
      rules: ['EQUALS','NOT_EQUALS','LIKE','NOT_LIKE','REGEX']
    },
    ATTR_KEY: {
      rules: ['EQUALS','NOT_EQUALS','LIKE','NOT_LIKE','REGEX']
    },
    ATTR_VAL: {
      rules: ['EQUALS','NOT_EQUALS','LIKE','NOT_LIKE','REGEX']
    }
  }
}

export const MUTATION = {
  targets: {
    NODE_NAME: {
      rules: ['ALL_REPLACE']
    },
    TEXT: {
      rules: ['START_OF','END_OF','ALL_REPLACE','REGEX_REPLACE']
    },
    ATTR_KEY: {
      rules: ['START_OF','END_OF','REGEX_REPLACE','ALL_REPLACE','CONCAT']
    },
    ATTR_VAL: {
      rules: ['START_OF','END_OF','REGEX_REPLACE','ALL_REPLACE','CONCAT']
    }
  }
}

export const defaultQuery = {
  type: 'QUERY',
  target: 'TEXT',
  targetValue: '',
  rule: 'EQUALS',
  ruleValue: '',
  ruleValueFlags: ''
}

export const defaultMutation = {
  type: 'MUTATION',
  target: 'TEXT',
  targetValue: '',
  rule: 'START_OF',
  ruleValue: '',
  ruleValueFlags: ''
}

export const defaultClause = {
  name: '',
  minimized: false,
  active: false,
  rules: [defaultQuery],
  view: []
}