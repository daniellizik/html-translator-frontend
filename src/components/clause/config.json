{
  "query": {
    "targets": {
      "nodeName": {
        "rules": [
          "equals",
          "notEquals",
          "like",
          "notLike"
        ]
      },
      "text": {
        "rules": [
          "equals",
          "notEquals",
          "like",
          "notLike",
          "regex"
        ]
      },
      "attrKey": {
        "rules": [
          "equals",
          "notEquals",
          "like",
          "notLike",
          "regex"
        ]
      },
      "attrVal": {
        "rules": [
          "equals",
          "notEquals",
          "like",
          "notLike",
          "regex"
        ]
      }
    },
    "defaultQuery": {
      "type": "query",
      "target": "text",
      "targetValue": "",
      "rule": "like",
      "ruleValue": "",
      "ruleValueFlags": ""
    }
  },

  "mutator": {
    "targets": {
      "nodeName": {
        "rules": [
          "allReplace"
        ]
      },
      "text": {
        "rules": [
          "startOf",
          "endOf",
          "allReplace",
          "regexReplace"
        ]
      },
      "attrKey": {
        "rules": [
          "startOf",
          "endOf",
          "regexReplace",
          "allReplace",
          "add"
        ]
      },
      "attrVal": {
        "rules": [
          "startOf",
          "endOf",
          "regexReplace",
          "allReplace",
          "add"
        ]
      }
    },
    "defaultMutation": {
      "type": "mutation",
      "target": "text",
      "targetValue": "",
      "rule": "startOf",
      "ruleValue": "",
      "ruleValueFlags": ""
    }
  }

}


