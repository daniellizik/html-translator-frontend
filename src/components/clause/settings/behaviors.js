// use as exceptions for mutations
// we need to be able to mutate text of child node
// flat list doesnt play nicely, so we need to do some 
// mapping via view ids

export const APPLY_TO_CHILDREN = (node, mutation, list, ruleResult) => {

}

export const APPLY_TO_NODE = (node, mutation, list, ruleResult) => {
  // console.log(node, ruleResult)
}