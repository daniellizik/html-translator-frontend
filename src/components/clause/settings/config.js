export const targets = ['NODE_NAME', 'TEXT', 'ATTR_KEY', 'ATTR_VAL']
export const behaviors = ['APPLY_TO_CHILDREN', 'APPLY_TO_NODE']

export const QUERY = {
  targets: {
    NODE_NAME: {
      rules: [
        'EQUALS', 
        'NOT_EQUALS', 
        'LIKE', 
        'NOT_LIKE',
        'REGEX'
      ],
      behaviors: []
    },
    TEXT: {
      rules: [
        'EQUALS', 
        'NOT_EQUALS', 
        'LIKE', 
        'NOT_LIKE',
        'REGEX'
      ],
      behaviors: []
    },
    ATTR_KEY: {
      rules: [
        'EQUALS', 
        'NOT_EQUALS', 
        'LIKE', 
        'NOT_LIKE',
        'REGEX',
        'HAS_NONE',
        'HAS_AT_LEAST'
      ],
      behaviors: []
    },
    ATTR_VAL: {
      rules: [
        'EQUALS', 
        'NOT_EQUALS', 
        'LIKE', 
        'NOT_LIKE',
        'REGEX'
      ],
      behaviors: []
    }
  }
}

export const MUTATION = {
  targets: {
    NODE_NAME: {
      rules: ['ALL_REPLACE', 'START_OF','END_OF'],
      behaviors
    },
    TEXT: {
      rules: ['ALL_REPLACE', 'START_OF', 'END_OF', 'REGEX_REPLACE'],
      behaviors
    },
    ATTR_KEY: {
      rules: ['ALL_REPLACE', 'START_OF', 'END_OF', 'REGEX_REPLACE'],
      behaviors
    },
    ATTR_VAL: {
      rules: ['ALL_REPLACE', 'START_OF', 'END_OF', 'REGEX_REPLACE'],
      behaviors
    }
  }
}

// hardcoded for parse5 ast spec
export const targetMap = {
  TEXT: 'value',
  NODE_NAME: 'nodeName'
}

export const defaultQuery = {
  targetValue: '',
  rule: 'EQUALS',
  ruleValue: '',
  ruleValueFlags: ''
}

export const defaultMutation = {
  targetValue: '',
  rule: 'START_OF',
  ruleValue: '',
  ruleValueFlags: '',
  behavior: ''
}

export const defaultClause = {
  name: '',
  minimized: false,
  active: false,
  target: 'TEXT',
  type: 'CLAUSE',
  queries: [defaultQuery],
  mutations: [],
  view: []
}