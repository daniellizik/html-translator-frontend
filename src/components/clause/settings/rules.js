import * as constants from '../constants'

export const EQUALS = ({before, targetValue}) => {
  if (!before || !targetValue)
    return false
  if (before === targetValue)
    return true
  else
    return false
}

export const NOT_EQUALS = ({before, targetValue}) => {
  if (!before || !targetValue)
    return false
  if (typeof before !== 'string' || typeof targetValue !== 'string')
    return false
  if (before !== targetValue)
    return true
  else
    return false
}

export const LIKE = ({before, targetValue}) => {
  if (!before || !targetValue)
    return false
  if (typeof before !== 'string' || typeof targetValue !== 'string')
    return false
  if (before.toLowerCase().indexOf(targetValue.toLowerCase()) > -1)
    return true
  else
    return false
}

export const NOT_LIKE = ({before, targetValue}) => {
  if (!before || !targetValue)
    return false
  if (before.toLowerCase().indexOf(targetValue.toLowerCase()) > -1)
    return false
  else
    return true
}

export const REGEX = ({before = '', ruleValue = '', ruleValueFlags = ''}) => {
  if (!before || !ruleValue)
    return false
  if (typeof before !== 'string' || typeof ruleValue !== 'string')
    return false
  if (ruleValueFlags && typeof ruleValueFlags !== 'string')
    return false
  let rx
  try {
    rx = new RegExp(ruleValue.replace(/\//g, '\\\\'), ruleValueFlags)
  }
  catch(e) {
    throw new Error(constants.CLAUSE_ERROR_INVALID_REGEX)
  }
  return rx.test(before)
}

export const HAS_NONE = ({before}) => {
  return before.length === 0
}

export const HAS_AT_LEAST = ({before, ruleValue}) => {
  return before.length >= ruleValue
}


// mutator  

export const ALL_REPLACE = ({ruleValue}) => {
  return ruleValue
}

export const START_OF = ({ ruleValue = '', before = '' }) => {
  return ruleValue + before
}

export const END_OF = ({ ruleValue = '', before = '' }) => {
  return before + ruleValue
}

export const REGEX_REPLACE = ({ruleValue = '', ruleValueFlags = '', targetValue = '', before = ''}) => {
  let rx
  try {
    rx = new RegExp((ruleValue).replace(/\//g, '\\\\'), ruleValueFlags)
  }
  catch(e) {
    throw new Error(constants.CLAUSE_ERROR_INVALID_REGEX)
  }
  return before.replace(rx, targetValue)
}

// implement these later...

export const ADD_ATTR = (base, ...attrs) => {
  return obj
}

export const REMOVE_ATTR = (obj) => {

}

export const REMOVE_ALL_ATTRS = (obj) => {

}
