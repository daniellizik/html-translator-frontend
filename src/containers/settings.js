import React, { Component } from 'react'
import { connect } from 'react-redux'

class Settings extends Component {
  constructor() {
    super()
  }
  componentDidMount() {

  }
  render() {
    return !this.props.viewing ? null : (
      <div class="container-fluid max p-0 m-0">
        
      </div>
    )
  }
}

const mapStateToProps = (state) => ({
  viewing: state.user.viewingSettings
})

export default connect(mapStateToProps)(Settings)