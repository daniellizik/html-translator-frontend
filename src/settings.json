{
  "onboarding_fade_delay": 700,
  "onboarding_fade_max": 3
}