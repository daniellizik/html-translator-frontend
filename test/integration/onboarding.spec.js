import { story, regression, step7regression } from '~/test/storyFixtures/onboarding'
import * as config from '~/src/components/clause/settings/config'

describe('onboarding', () => {

  it('should be at the correct step with overlay display', () => {
    story.forEach(s => {
      expect({
        onboarding: s.onboarding,
        overlay: s.overlay
      }).toMatchSnapshot()
    })
  })

  it('should have correct clauses', () => {
    story.forEach(s => {
      expect(s.clauses).toMatchSnapshot()
    })
  })

  it('should have correct source', () => {
    story.forEach(s => {
      expect(s.source).toMatchSnapshot()
    })
  })

  it('should have correct xml', () => {
    story.forEach(s => {
      expect(s.xml).toMatchSnapshot()
    })
  })

})