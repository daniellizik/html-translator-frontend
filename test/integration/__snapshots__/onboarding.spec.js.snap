exports[`onboarding should be at the correct step with overlay display 1`] = `
Object {
  "onboarding": Object {
    "step": 0,
  },
  "overlay": true,
}
`;

exports[`onboarding should be at the correct step with overlay display 2`] = `
Object {
  "onboarding": Object {
    "step": 1,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 3`] = `
Object {
  "onboarding": Object {
    "step": 2,
  },
  "overlay": true,
}
`;

exports[`onboarding should be at the correct step with overlay display 4`] = `
Object {
  "onboarding": Object {
    "step": 3,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 5`] = `
Object {
  "onboarding": Object {
    "step": 4,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 6`] = `
Object {
  "onboarding": Object {
    "step": 5,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 7`] = `
Object {
  "onboarding": Object {
    "step": 6,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 8`] = `
Object {
  "onboarding": Object {
    "step": 7,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 9`] = `
Object {
  "onboarding": Object {
    "step": 7,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 10`] = `
Object {
  "onboarding": Object {
    "step": 7,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 11`] = `
Object {
  "onboarding": Object {
    "step": 7,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 12`] = `
Object {
  "onboarding": Object {
    "step": 10,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 13`] = `
Object {
  "onboarding": Object {
    "step": 10,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 14`] = `
Object {
  "onboarding": Object {
    "step": 10,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 15`] = `
Object {
  "onboarding": Object {
    "step": 10,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 16`] = `
Object {
  "onboarding": Object {
    "step": 10,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 17`] = `
Object {
  "onboarding": Object {
    "step": 10,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 18`] = `
Object {
  "onboarding": Object {
    "step": 10,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 19`] = `
Object {
  "onboarding": Object {
    "step": 10,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 20`] = `
Object {
  "onboarding": Object {
    "step": 10,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 21`] = `
Object {
  "onboarding": Object {
    "step": 10,
  },
  "overlay": false,
}
`;

exports[`onboarding should be at the correct step with overlay display 22`] = `
Object {
  "onboarding": Object {
    "step": 10,
  },
  "overlay": false,
}
`;

exports[`onboarding should have correct clauses 1`] = `Array []`;

exports[`onboarding should have correct clauses 2`] = `Array []`;

exports[`onboarding should have correct clauses 3`] = `Array []`;

exports[`onboarding should have correct clauses 4`] = `Array []`;

exports[`onboarding should have correct clauses 5`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [],
    "target": "TEXT",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 6`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [],
    "target": "TEXT",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 7`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "",
      },
    ],
    "target": "TEXT",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 8`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "",
      },
    ],
    "target": "TEXT",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 9`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "",
      },
    ],
    "target": "NODE_NAME",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 10`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "tr",
      },
    ],
    "target": "NODE_NAME",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 11`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "tr",
      },
    ],
    "target": "NODE_NAME",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 12`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "tr",
      },
    ],
    "target": "NODE_NAME",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 13`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "tr",
      },
    ],
    "target": "NODE_NAME",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 14`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "tr",
      },
    ],
    "target": "NODE_NAME",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 15`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "tr",
      },
    ],
    "target": "NODE_NAME",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 16`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "tr",
      },
    ],
    "target": "NODE_NAME",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 17`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "tr",
      },
    ],
    "target": "NODE_NAME",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 18`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "tr",
      },
    ],
    "target": "NODE_NAME",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 19`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "tr",
      },
    ],
    "target": "NODE_NAME",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 20`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "tr",
      },
    ],
    "target": "NODE_NAME",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 21`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "tr",
      },
    ],
    "target": "NODE_NAME",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct clauses 22`] = `
Array [
  Object {
    "active": false,
    "minimized": false,
    "mutations": Array [],
    "name": "",
    "queries": Array [
      Object {
        "rule": "EQUALS",
        "ruleValue": "",
        "ruleValueFlags": "",
        "targetValue": "tr",
      },
    ],
    "target": "NODE_NAME",
    "type": "CLAUSE",
    "view": Array [],
  },
]
`;

exports[`onboarding should have correct source 1`] = `
Object {
  "active": false,
  "lastModified": null,
  "name": null,
  "rawHtml": null,
  "url": null,
}
`;

exports[`onboarding should have correct source 2`] = `
Object {
  "active": false,
  "lastModified": null,
  "name": null,
  "rawHtml": null,
  "url": null,
}
`;

exports[`onboarding should have correct source 3`] = `
Object {
  "active": true,
  "lastModified": "url",
  "name": null,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 4`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 5`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 6`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 7`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 8`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 9`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 10`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 11`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 12`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 13`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 14`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 15`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 16`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 17`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 18`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 19`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 20`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 21`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct source 22`] = `
Object {
  "active": false,
  "lastModified": "url",
  "name": undefined,
  "rawHtml": null,
  "url": "sample-email.html",
}
`;

exports[`onboarding should have correct xml 1`] = `undefined`;

exports[`onboarding should have correct xml 2`] = `undefined`;

exports[`onboarding should have correct xml 3`] = `undefined`;

exports[`onboarding should have correct xml 4`] = `undefined`;

exports[`onboarding should have correct xml 5`] = `undefined`;

exports[`onboarding should have correct xml 6`] = `undefined`;

exports[`onboarding should have correct xml 7`] = `undefined`;

exports[`onboarding should have correct xml 8`] = `undefined`;

exports[`onboarding should have correct xml 9`] = `undefined`;

exports[`onboarding should have correct xml 10`] = `undefined`;

exports[`onboarding should have correct xml 11`] = `undefined`;

exports[`onboarding should have correct xml 12`] = `undefined`;

exports[`onboarding should have correct xml 13`] = `undefined`;

exports[`onboarding should have correct xml 14`] = `undefined`;

exports[`onboarding should have correct xml 15`] = `undefined`;

exports[`onboarding should have correct xml 16`] = `undefined`;

exports[`onboarding should have correct xml 17`] = `undefined`;

exports[`onboarding should have correct xml 18`] = `undefined`;

exports[`onboarding should have correct xml 19`] = `undefined`;

exports[`onboarding should have correct xml 20`] = `undefined`;

exports[`onboarding should have correct xml 21`] = `undefined`;

exports[`onboarding should have correct xml 22`] = `undefined`;
